src/Sentry/SentryLaravel/SentryLaravelServiceProvider.php

```diff
@@ -86,16 +86,18 @@ public function register()
                 'app_path' => app_path(),
             ), $user_config));
 
-            // bind user context if available
-            try {
-                if ($app['auth']->check()) {
-                    $user = $app['auth']->user();
-                    $client->user_context(array(
-                        'id' => $user->getAuthIdentifier(),
-                    ));
+            if($user_config['breadcrumbs.user_context']) {
```

Comment:
we wouldnt want to call this breadcrumbs -- not sure if 'user_context' is descriptive enough.

maybe user_context.auto = True/False?